### PR TITLE
chore: add translations for API screens

### PR DESCRIPTION
**Pull request by timarney**
# Summary | Résumé

PR to add translations for the API screens.

### CODE REVIEW

**Comment by timarney** [comment]
@anikbrazeau starting this PR for when you have some time to start adding the strings

There's a bunch on the form-builder file 

i.e. 

    "responseDeliveryHelpWithApi": {

File: public/static/api/Readme.md [hunk 1]
```diff
@@ -1,7 +1,3 @@
 # API Readme
 
-This document provides an overview of the API endpoints available in the platform-forms-client project.
-
-## Contact
-
-For any questions or support, please contact the API team at [support@example.com](mailto:support@example.com).
+Update text
```
Comment (timarney) [inline public/static/api/Readme.md:3]: Here's the Readme file for the .zip API key download

**Comment by anikbrazeau** [comment]
Added a couple of commits to the form builder file to update the French strings and match some tweaks in English as well!

**Comment by anikbrazeau** [comment]
Created a first bilingual version of the READ.ME that we can tweak later as well.

**Review by dsamojlenko — APPROVED** [review]

**Pull request merged**